packages/cozy-client/src/store/queries.js

```diff
@@ -129,6 +129,9 @@ const updateData = (query, newData) => {
 }
 
 const autoQueryUpdater = action => query => {
+  if (!action.response || !action.response.data) {
+    return query
+  }
```

Comment:
When does this happen? If it's when the server returns an error, is there something else that we can do aside from not updating the query?

Reply:
I am going to check this, thanks

Reply:
The relevant query state will be in error since it will go through the `query` reducer and not the `autoQueryUpdater`.